ant-cli/src/commands.rs

```diff
@@ -228,7 +228,7 @@ pub async fn handle_subcommand(opt: Opt) -> Result<()> {
                 password,
             } => wallet::import(private_key, no_password, password),
             WalletCmd::Export => wallet::export(),
-            WalletCmd::Balance => wallet::balance().await,
+            WalletCmd::Balance => wallet::balance(peers.await?.is_local()).await,
```

Comment:
Not really fond of this hack, but couldn't find an easier way to see if we are running locally?

Reply:
Yeah, since we don't need peers for balance it's kind of a waste but it'll do!